cmd/context/utils.go

```diff
@@ -186,12 +186,43 @@ func LoadStackWithContext(ctx context.Context, name, namespace string, stackPath
 	return s, nil
 }
 
+//LoadManifestV2WithContext initializes the okteto context taking into account command flags and manifest namespace/context fields
+func LoadManifestV2WithContext(ctx context.Context, namespace, path string) error {
+	ctxOptions := &ContextOptions{
+		Namespace: namespace,
+		Show:      true,
+	}
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	manifest, err := GetManifestV2(cwd, path)
+	if err != nil {
+		if err != errors.ErrManifestNotFound {
+			return err
+		}
+	} else {
+		ctxOptions.Context = manifest.Context
+		if ctxOptions.Namespace == "" {
+			ctxOptions.Namespace = manifest.Namespace
+		}
+	}
+
+	if err := Run(ctx, ctxOptions); err != nil {
+		return err
+	}
+
+	return nil
+}
+
 // GetManifest Loads a manifest
 func GetManifest(ctx context.Context, srcFolder string, opts ManifestOptions) (*model.Manifest, error) {
 	pipelinePath := getPipelinePath(srcFolder, opts.Filename)
 	if pipelinePath != "" {
 		log.Debugf("Found okteto manifest %s", pipelinePath)
-		manifest, err := LoadManifestWithContext(ctx, pipelinePath, opts.Namespace, opts.K8sContext)
+		manifest, err := utils.LoadManifest(pipelinePath)
```

Comment:
This function might belong better with a good location for the new function above? I don't know how many users of LoadManifest there are but if it's a small number it's probably worth the move. Why is loading of manifest separated between so many packages that are focused on types, interfaces or are generic? How would someone easily begin to read code associated with manifests and could they do it with a simple text editor?